RAKE/FoxStopList.py

```diff
@@ -0,0 +1,430 @@
+list=['a',
```

Comment:
list is a reserved keyword in python, can be dangerous to collide with it. Recommend you rename it to `stopwords` or something just to avoid unwanted side-effects